src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+const STATUS = 'book-store/redux/book/bookSlice/STATUS';
+
+const defultState = '';
+export const categoryReducer = (state = defultState, action) => {
+  switch (action.type) {
+    case STATUS:
+      return 'UNDER  CONSTRUCTION';
+    default:
+      return state;
+  }
+};
+
+export const Status = () => ({
+  type: STATUS,
+});
```

Comment:
- [ ] Similarly, _categoriesSlice_ to make use of `createSlice` from `@reduxjs/toolkit`, and should have the following implementations:
  * Imports `createSlice` from `@reduxjs/toolkit`
  * Creates an `initialState` with a state value `categories` that contains an empty array
  * Creates 1 reducer, `checkStatus`
  * Exports by default `categoriesSlice.reducer`
  * Exports by `checkStatus` as an action

It should resemble the following code example:
```javascript
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```